docs/ServingTheServer.md

```diff
@@ -0,0 +1,138 @@
+# Linking Your Dev Server to Slack
+
+So, you want to see some progress, preferably on slack, and not just in the
+forms of unit testing? At this point, fear is actually a reasonable response.
+With this guide, you can be talking to your locally-hosted slack bot in no time!
+
+## 1: Registering Your Slack Bot
+
+First things first, you must first create a slack workspace (an existing one
+would do, as long as you can install things onto it).
+
+After that, you must create a new slack app. Follow the link
+[here][make-slack-app] to create a new slack app. Make sure to have it installed
+onto your slack workspace of choice.
+
+Now, you need to create a bot user. [Create it here][make-bot].
+
+## 2: Download and Start Up DynamoDB
+
+Since we are developing locally, we must use the local version of DynamoDB.
+Download the `jar` from [here][dynamodb-download]. Follow the instructions from
+another tab to have it up and running properly.
+
+## 3: Download and Install (and run) ngrok
+
+Slack does not run locally. To force/trick it to run locally (method recommended
+by slack), you must forward your 5000 port to a public domain name. Luckily for
+us, `ngrok` can do such things. Sign up and download and run `ngrok`
+[here][download-ngrok].
+
+Our app runs on port 5000. Run `ngrok http 5000` and take note of the forwarding
+URL (this means copy it).
```

Comment:
I think you could just say "copy the forwarding URL" here, instead of the parenthesized aside.